src/web/package.json

```diff
@@ -22,19 +21,23 @@
     "jwt-decode": "^3.1.2",
     "nanoid": "^3.1.22",
     "next": "^10.1.3",
+    "next-compose-plugins": "^2.2.1",
+    "next-pwa": "^5.1.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-use": "^17.2.1",
     "smoothscroll-polyfill": "^0.4.4",
     "swr": "^0.5.5",
-    "valid-url": "^1.0.9",
     "yup": "^0.32.9"
   },
   "devDependencies": {
     "@testing-library/react": "^11.2.6",
+    "@mdx-js/loader": "^1.6.22",
+    "@mdx-js/mdx": "^1.6.22",
     "@types/node": "^14.14.37",
     "@types/react": "^17.0.3",
-    "eslint-plugin-jest": "^24.3.4",
+    "babel-loader": "^8.2.2",
+    "terser-webpack-plugin": "^5.1.1",
```

Comment:
Where is this used?

Reply:
When I implemented `next-pwa` and using webpack 5, there was an error with webpack saying that `terser-webpack-plugin` cannot be found. So I installed it as a devDep and the error gone.

Reply:
It is for sth used inside `next-pwa`